xls/dslx/lsp/language_server_adapter.cc

```diff
@@ -421,4 +421,39 @@ LanguageServerAdapter::Rename(std::string_view uri,
   return std::nullopt;
 }
 
+absl::StatusOr<std::vector<verible::lsp::DocumentHighlight>>
+LanguageServerAdapter::DocumentHighlight(
+    std::string_view uri, const verible::lsp::Position& position) const {
+  if (ParseData* parsed = FindParsedForUri(uri); parsed && parsed->ok()) {
+    FileTable& file_table = parsed->file_table();
+    const Pos pos = ConvertLspPositionToPos(uri, position, file_table);
+    VLOG(1) << "FindDefinition; uri: " << uri << " pos: " << pos;
+    const Module& module = parsed->module();
+    std::optional<const NameDef*> maybe_definition = xls::dslx::FindDefinition(
+        module, pos, parsed->type_info(), parsed->import_data());
+    if (maybe_definition.has_value()) {
+      const NameDef* name_def = maybe_definition.value();
+      const Span& definition_span = name_def->span();
+      std::vector<verible::lsp::DocumentHighlight> highlights = {
+          verible::lsp::DocumentHighlight{
+              .range = ConvertSpanToLspRange(definition_span),
+              // TODO(cdleary): 2024-10-13 How do we indicate the kind?
+              //.kind = verible::lsp::DocumentHighlightKind::Write,
```

Comment:
this comment (and below in 450) look like the want a space between `//` and `.kind` according to the `clang-format` gods.... :)

Reply:
Makes sense, just removed for the time being.